### PR TITLE
Events/wayoflife

### PR DESCRIPTION
**Pull request by Whizzer**
All restructured events were tested. Events most likely to still contain errors are in wol_lover_events and wol_seduction_events.

### CODE REVIEW

**Comment by Whizzer** [comment]
> > 
> 
> ```
> 	is_within_diplo_range = ROOT
> 
> 	primary_title = {
> 			is_landless_type_title = no
> 	}
> 
> 	 NOR = {
> 			is_rival = ROOT
> 			is_liege_or_above_of = ROOT
> 	 }
> }
> 
> preferred_limit = {
> 		NOT = { religion_group = ROOT }
> 		is_within_diplo_range = ROOT
> }
> 
> preferred_limit = {
> 	NOT = { religion = ROOT }
> 	is_within_diplo_range = ROOT
> }
> ```
> 
> `is_within_diplo_range` is currently in both the limit and the preferred_limit here (as well as the the tombola at top). Should probably be one or the other.

Yes, it should be removed from the limit, with another preferred_limit containing only 'is_within_diplo_range' added last.

File: CleanSlate/events/wol_business_events.txt [hunk 1]
```diff
@@ -1062,13 +1421,13 @@ character_event = {
 # Steward conclusion
 character_event = {
 	id = WoL.10127
+
 	is_triggered_only = yes
-	hide_window = yes ###fix if time
+	hide_window = yes # TODO: Fix if time
```
Comment (schwarherz) [inline CleanSlate/events/wol_business_events.txt:1426]: What needs fixed, exactly?

**Review by schwarherz — CHANGES_REQUESTED** [review]
I made note of a few questions I have. Otherwise looks good.

File: CleanSlate/events/mnm_assassins_events.txt [hunk 2]
```diff
@@ -179,6 +192,28 @@ letter_event = {
...
 	}
+
+	after = {
+		if = {
+			limit = { has_character_modifier = assassins_debt }
+
+			multiply_variable = {
+				which = twice_assassins_debt
+				which = assassins_debt
+			}
```
[the remark this answers is not in the document]
Comment (SuccinctScrivener) [reply CleanSlate/events/mnm_assassins_events.txt:203]: Its used in the ai_will_do of the decision, and we can't manipulate the variable there.

File: CleanSlate/events/wol_business_events.txt (hunk 1 above)
Comment (SuccinctScrivener) [reply CleanSlate/events/wol_business_events.txt:1426]: I'm assuming the developer intended to add localisation and make the event visible, as the steward could be a player. Looking at it again, if that's not something we're adding in, we could just remove WoL.10127 entirely.

File: CleanSlate/events/republic_trade_events.txt [hunk 3]
```diff
@@ -22,20 +22,21 @@ character_event = {
...
+		trigger_if = {
+			limit = { is_merchant_republic = yes }
 
+			any_vassal = {
+				is_patrician = yes
+				ai = yes
+			}
+		}
+		trigger_else = {
+			liege = {
```
[the remark this answers is not in the document]
Comment (SuccinctScrivener) [reply CleanSlate/events/republic_trade_events.txt:34]: It shouldn't. Make sure you're paying attention to which scope the event is firing from.

**Review by SuccinctScrivener — DISMISSED** [review]
Looks good.

My only general comment is that I think we've been a bit overzealous in removing hidden_effect blocks overall, removing them in places where they don't suppress a tooltip but a portrait or trait icon on an event option. This is the one area where a hidden_effect isn't fully redundant with custom_tooltip.

I also came across a bug in my testing, but I can't see a reason why. I haven't checked yet if the issue is present in vanilla. When I tried to start a party where only one person had accepted the invitation to carousing , I would randomly sometimes get a "nobody accepted your invitation" event (WoL.4015) and no party, and the invited person would be stuck with the do not disturb flag.

**Comment by Whizzer** [comment]
> Looks good.
> 
> My only general comment is that I think we've been a bit overzealous in removing hidden_effect blocks overall, removing them in places where they don't suppress a tooltip but a portrait or trait icon on an event option. This is the one area where a hidden_effect isn't fully redundant with custom_tooltip.
> 
> I also came across a bug in my testing, but I can't see a reason why. I haven't checked yet if the issue is present in vanilla. When I tried to start a party where only one person had accepted the invitation to carousing , I would randomly sometimes get a "nobody accepted your invitation" event (WoL.4015) and no party, and the invited person would be stuck with the do not disturb flag.

Good find. I tested this before, but clearly not enough. The commit below does fix it.
You may be right on the suppression of tooltips, but I would prefer that is tested to be sure.

**Review by schwarherz — APPROVED** [review]

**Pull request merged**